openstack/designate/templates/etc-configmap.yaml

```diff
@@ -25,3 +25,8 @@ data:
 {{ include (print .Template.BasePath "/etc/_designate-policy.json.tpl") . | indent 4 }}
   logging.conf: |
 {{ include "loggerIni" .Values.logging | indent 4 }}
+
+{{- if .Values.audit.enabled }}
+    designate_audit_map.yaml: |
+{{ include (print .Template.BasePath "/etc/_designate_audit_map.yaml") . | indent 4 }}
```

Comment:
how is this file copied to /etc/designate?

Reply:
added copy now, without the template audit-enabled call, as the file is not a template right now.